sql/item_usable.sql

```diff
@@ -2341,6 +2342,7 @@ INSERT INTO `item_usable` VALUES (21266,'gastraphetes',1,1,55,0,1,10,3600,0);
 INSERT INTO `item_usable` VALUES (21267,'annihilator',1,1,55,0,1,10,3600,0);
 INSERT INTO `item_usable` VALUES (21268,'death_penalty',1,1,55,0,1,10,3600,0);
 INSERT INTO `item_usable` VALUES (21269,'armageddon',1,1,55,0,1,10,3600,0);
+INSERT INTO `item_usable` VALUES (21868,'sha_wujings_la._+1',1,3,0,0,1,30,72000,0); -- Dispenses: Distilled Water x 12 (TODO: Verify animation)
```

Comment:
> 4:51 PM]Carver: sir, do you have this item by any chance?
Image
[4:52 PM]Carver: idk the animation for this, don't think you'd approve me guessing based on past experiences 😄
[6:10 PM]TeoTwawki: I do have one. its the generic item use animation
[6:11 PM]TeoTwawki: scratch butt raise hand item get messaging

Reply:
😄